Furniture/Shelf/IKEA_Kallax_77x42cm.step

```diff
@@ -0,0 +1,1931 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('FreeCAD Model'),'2;1');
+FILE_NAME(
+'/home/pablo/Workspace/FreeCAD/FreeCAD-library/Furniture/Shelf/IKEA_
```

Comment:
Is it possible to remove the path in  `FILE_NAME()`

Reply:
Done.